src/main/java/org/embulk/output/databricks/DatabricksAPIClient.java

```diff
@@ -11,9 +12,18 @@
 
 public class DatabricksAPIClient {
   public static DatabricksAPIClient create(DatabricksOutputPlugin.DatabricksPluginTask task) {
+    SetUserAgent(task);
+
     return new DatabricksAPIClient(createDatabricksConfig(task));
   }
 
+  private static void SetUserAgent(DatabricksOutputPlugin.DatabricksPluginTask task) {
```

Comment:
普通にメソッドなので先頭は小文字でいい気がします！僕の知らないお作法だったらごめんなさい！

```suggestion
  private static void setUserAgent(DatabricksOutputPlugin.DatabricksPluginTask task) {
```